internal/providers/vultr/vultr.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The vultr provider fetches a remote configuration from the vultr
+// user-data metadata service URL.
+// https://web.archive.org/web/20190513194756/https://www.vultr.com/metadata/#user
```

Comment:
This link says that user data is under `/user-data/user-data`, not `latest/user-data` – are both correct?

Reply:
Yeah, that's also something already upstream: https://github.com/coreos/ignition/blob/master/internal/providers/vultr/vultr.go
That uses `/user-data/user-data`